pubmed_dl/bin/pubmed-dl

```diff
@@ -43,7 +44,8 @@ def main(start_date: str, end_date: str, request_file: str):
             for batch in uids_to_docs(pmids):
                     for doc in batch:
                         f.write(f"{json.dumps(doc)}\n")
-                        progress.update(1)
+                        if progress_bar is True:
+                            progress.update(1)
```

Comment:
Doesn't the progress bar still show up if `not progress_bar`, but never updated?

Reply:
i ran it like this, and it never showed up